src/screens/discover/components/DiscoverHome.js

```diff
@@ -45,38 +49,42 @@ export default function DiscoverHome() {
     ).length > 0;
 
   return (
-    <Inset top="20px" bottom={{ custom: 150 }}>
-      <Stack space="20px">
-        <Inset horizontal="20px">
-          {!testNetwork ? (
-            <Stack space="20px">
-              <Inline space="20px">
-                <GasCard />
-                {isProfilesEnabled && <ENSSearchCard />}
-              </Inline>
-              {nftOffersEnabled && <NFTOffersCard />}
-              {/* We have both flags here to be able to override the remote flag and show the card anyway in Dev*/}
-              {(opRewardsRemoteFlag || opRewardsLocalFlag) && <OpRewardsCard />}
-              {hardwareWalletsEnabled && !hasHardwareWallets && <LedgerCard />}
-              {isProfilesEnabled && <ENSCreateProfileCard />}
-              <Inline space="20px">
-                <LearnCard cardDetails={backupsCard} type="square" />
-                <LearnCard cardDetails={avoidScamsCard} type="square" />
-              </Inline>
-              <DPICard />
-            </Stack>
-          ) : (
-            <Stack space="20px">
-              <Inline space="20px">
-                <GasCard />
-                <LearnCard cardDetails={cryptoAndWalletsCard} type="square" />
-              </Inline>
-              <DPICard />
-            </Stack>
-          )}
-        </Inset>
-        <Lists />
-      </Stack>
+    <Inset top="10px" bottom={{ custom: 150 }}>
```

Comment:
moved vertical padding applied on each Card OUT of DiscoverHome and IN to the individual Card components.

this makes it possible to conditionally render Cards while still keeping this rendering logic inside the respective Card components instead of in DiscoverHome. that probably sounds confusing but hmu if you want a better explanation